ironfish-cli/scripts/import-export-test-vector/0p1p65_blob.txt

```diff
@@ -0,0 +1 @@
+ironfishaccount0000010v38vetjwd5k7m3z8gcjcgnwv9kk2g36ygc8qvtsxc647cnvda3zytpzwdcx2mnyd9hxwjm90y3r5g34xa3nswpkvf3xzwrzvejnvd33vcekzen9x9nx2v3exq6xyvmyvgckzvn9xvmrqen9xqmn2cnrv4nxzef5xgenydfkve3n2vp5vgcr2g3vyfmxjethfdjhjg36ygmnzwfevv6kxwryxejrvcenxqurjce3xymnqvenvf3xzcmzxuerzetzxsurvc3sxgur2vfhx9snyvp3xguxvvtrvdjngvpkxc6rwve4xuckvwfhxu6rwctzxycnqvnxx56xydfnxp3xgefj89jk2wp4xv6kvwpkxdjrwdf4vdskvdnxx4nxvdp5v5erydmrvymrwenr89jnqc3z9s3xjmnrdakkjmn82e5k2a6tv4ujyw3zxp3nxve5v5uxzenrvgmkzvpcv93nyvf4xcmnwd3cvdjxyetr8y6kye3cxyergvf5vc6ngdmrvyunvvnyxycn2efc8qekgwtxvgenqvpz9s3x7at5vahkjmn82e5k2a6tv4ujyw3zvfsn2wt9x93nxc3hvvenywtzxq6n2vp3xp3n2enrxgux2de4x4jx2v3hxyunjd33vvuryv3nvscrwen9vgervvpev3nrwenx8qckxv3z9s38qatzd35kxstyv3ex2umnygazydpexejngdm9vserxctxvejkvd35x43kxe3jv33xxef4vg6rsdeevfjnwep3xgcxyenxxycxzc34x9snswp5vfnxgven8qcnywf589jzylg7sfvp6
```

Comment:
We usually call these fixtures in our code base, it would be nice to remove the suffix or call them fixtures. It'd be fine to leave the suffix off too.